diff.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+
+def compare_source_files_objects(obj1, obj2):
+    diff_funcs = []
+    if obj1['name'] == obj2['name']:
+        if obj1['functions'] != obj2['functions'] or obj1['coverage'] != obj2['coverage']:
+            if obj1['functions'] == obj2['functions']:
+                obj1['functions'] = []
+            if obj1['functions'] != obj2['functions']:
+                for func1 in obj1['functions']:
+                    for func2 in obj2['functions']:
+                        if func1['name'] == func2['name']:
+                            if func1['exec'] is False and func2['exec'] is True:
+                                func1['exec'] = True
+                                diff_funcs.append(func1)
```

Comment:
If you do `diff_funcs.append(func2)` you don't need the previous `func1['exec'] = True` line.